suggest/acronyms/gen_acros.py

```diff
@@ -0,0 +1,52 @@
+"""Script to generate simple acronyms based on first character of each word in a sentence.
```

Comment:
nit on filename: I tend to prefer full names like generate_acronyms over abbreviations. While I understood from context, gen_acros could be confusing out of context.